### ISSUE TITLE
Version support

### ISSUE THREAD

**Issue by ElectricPrism** [goldeneye-source/ges-python#5]
Is this information in README.md still current?

The current version of GE:S these libraries support is the unreleased v4.2.3. They are mostly backwards compatible to v4.2.2 however.

I also noticed there's no CMake, would that be something to add to the next update of this?

**Comment by droidmonkey** [comment]
Good catch, the current version this python code supports is v5.0. There is no cmake as this is interpreted by the game as its running (ie no compiling).
